util/BUILD

```diff
@@ -4,3 +4,7 @@ py_binary(
     name = "config_stripper",
     srcs = ["config_stripper.py"],
 )
+
+exports_files(["commit.sh.tpl"])
```

Comment:
nit, could you merge these two into one list?

Reply:
done